frontend/wallet/src/main/GraphqlMain.re

```diff
@@ -0,0 +1,20 @@
+open Tc;
+let createClient = uri => {
+  let inMemoryCache = ApolloInMemoryCache.createInMemoryCache();
+  let httpLink =
+    ApolloLinks.createHttpLink(~uri, ~fetch=Bindings.Fetch.fetch, ());
+  ReasonApollo.createApolloClient(~link=httpLink, ~cache=inMemoryCache, ());
```

Comment:
ah we can just replace the link here to get the cross-rpocess thing. Doesn't seem so bad... Do you think _not_ doing it sooner will get us a lot of code we'll need to throw away later? It doesn't seem like it to me, so this is probably fine for now, but worth thinking about.